src/components/graph/Graph.tsx

```diff
@@ -127,26 +131,24 @@ function Graph() {
 
 	return (
 		<div className='graph' ref={rootRef}>
-			{!activeWorkspace.isSearchWorkspace && (
-				<GraphChunksVirtualizer
-					chunkWidth={chunkWidth}
-					settings={settings}
-					renderChunk={renderChunk}
-					setRange={activeWorkspace.graphStore.setRange}
-					getChunk={getChunk}
-					interval={activeWorkspace.graphStore.interval}
-					timestamp={activeWorkspace.graphStore.timestamp}
-					key={activeWorkspace.graphStore.timestamp}
-					panelsRange={panelsRange}
-					range={activeWorkspace.graphStore.range}
-				/>
-			)}
+			<GraphChunksVirtualizer
+				chunkWidth={chunkWidth}
+				settings={settings}
+				renderChunk={renderChunk}
+				setRange={activeWorkspace.graphStore.setRange}
+				getChunk={getChunk}
+				interval={activeWorkspace.graphStore.interval}
+				timestamp={activeWorkspace.graphStore.timestamp}
+				key={activeWorkspace.graphStore.timestamp}
+				panelsRange={panelsRange}
+				range={activeWorkspace.graphStore.range}
+			/>
 			<GraphOverlay
```

Comment:
I dont know if GraphOverlay component make sence now since all previous markup and components are gone. Maybe we should move outside utems components to Graph